### PR TITLE
Server chore: Updated Level and Subjects Enum

### LINKED ISSUE #41: Decide what to include in the level and subject enums

### PR DESCRIPTION
**Pull request by yiyoupteltd**
close #41

### CODE REVIEW

**Review by EugeneTeu — DISMISSED** [review]

File: server/src/utils/constants/Level.ts [hunk 1]
```diff
@@ -1,6 +1,16 @@
 export enum Level {
-  PRIMARY = "primary",
-  SECONDARY = "secondary",
-  JUNIOR_COLLEGE = "junior college",
-  DEFAULT = "others",
+  //ranked in increasing age
+  PRIMARY = "Primary",
+  SECONDARYEXPRESS = "Secondary Express",
+  SECONDARYNA = "Secondary Normal Academic",
+  SECONDARYNT = "Secondary Normal Technical",
```
Comment (AaronCQL) [inline server/src/utils/constants/Level.ts:6]: Let's put an `_` between the `secondary` and the next word for better readability.

**Review by AaronCQL — CHANGES_REQUESTED** [review]

**Comment by EugeneTeu** [comment]
@yiyoupteltd pls do the change

**Comment by yiyoupteltd** [comment]
> @yiyoupteltd pls do the change

Ps doing it now, thanks for the reminder

**Comment by EugeneTeu** [comment]
yiyoupteltd dismissed EugeneTeu’s stale review via 3b69047 39 seconds ago
sad

**Review by EugeneTeu — APPROVED** [review]

**Pull request merged**